internal/backend/sessions/sessionworkflows/workflow.go

```diff
@@ -83,15 +88,17 @@ func runWorkflow(
 		signals: make(map[string]uint64),
 	}
 
-	if err = w.initEnvModule(ctx); err != nil {
+	var cinfos map[string]connInfo
```

Comment:
Map is not initialised, is this intentional?

Reply:
yup, don't want to overwrite err in the next line